crates/ruff_linter/src/rules/refurb/rules/single_item_membership_test.rs

```diff
@@ -78,8 +79,9 @@ pub(crate) fn single_item_membership_test(
         _ => return,
     };
 
-    // Check if the right-hand side is a single-item object.
-    let Some(item) = single_item(right) else {
+    // Check if the right-hand side is a single-item object
+    let semantic = checker.semantic();
+    let Some(item) = single_item(right, semantic) else {
```

Comment:
Nit: I'd just leave it like this to minimize the diff. `semantic` isn't used anywhere else anyway.

```suggestion
    // Check if the right-hand side is a single-item object.
     let Some(item) = single_item(right, checker.semantic()) else {
```